### PR TITLE
[move-prover] added informal specs for fixedpoint32.move and lbr.move

### PR DESCRIPTION
**Pull request by junkil-park**
- added the informal specs for both modules FixedPoint32 and LBR

## Motivation

To specify the standard library

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

cargo test

### CODE REVIEW

File: language/stdlib/modules/fixedpoint32.move [hunk 1]
```diff
@@ -74,6 +74,95 @@ module FixedPoint32 {
...
+    * Let FP0 refer to u64 which can be seen as the 64-bit unsigned fixed-point numeric type with 0 factional bits.
+    * round_FP32: R -> FP32 is a function that takes a real number as an input and returns the largest FP32 number which
+      is less than or equal to the input.
+    * round_FP0: R -> FP0 is a function that takes a real number as an input and returns the largest FP0 number which is
+      less than or equal to the input.
+    * Let mul_R and div_R be the multiplication and the division function over real numbers respectively.
+    */
+
+    spec fun multiply_u64 {
+        // aborts_if mul_R(num, multiplier) >= max_u64() + 1; // overflow (note that max_u64() == max_FP0())
```
Comment (DavidLDill) [inline language/stdlib/modules/fixedpoint32.move:104]: This is a nitpick, but wouldn't > max_u64() be more clear than >= max_u64() + 1?

File: language/stdlib/modules/fixedpoint32.move [hunk 2]
```diff
@@ -74,6 +74,95 @@ module FixedPoint32 {
...
+        // aborts_if 0 < mul_R(num, multiplier) < 1; // underflow (note that 1 is the smallest non-zero FP0);
```
Comment (DavidLDill) [inline language/stdlib/modules/fixedpoint32.move:105]: Shouldn't the result just be 0 in this case?

File: language/stdlib/modules/fixedpoint32.move (hunk 1 above)
Comment (junkil-park) [reply language/stdlib/modules/fixedpoint32.move]: They are subtly different. (max_u64()+0.1) can be rounded down to max_u64(), but the integer that (max_u64()+1.1) is rounded down to is not u64. So, the latter is definitely an overflow.

File: language/stdlib/modules/fixedpoint32.move (hunk 2 above)
Comment (junkil-park) [reply language/stdlib/modules/fixedpoint32.move]: This is a condition for underflow in general. We may need to abort if it's not a big deal for us. However, it seems that the other function called `create_from_rational` actually cares about the underflow case.

**Pull request closed without merging**